tests/integration/_experimental/object_detection/test_evaluator_single_class_f1_optimal.py

```diff
@@ -272,7 +272,8 @@ def test__object_detection__multiclass_evaluator__f1_optimal() -> None:
     assert config.display_name() in eval.evaluator.threshold_cache
     assert eval.evaluator.threshold_cache[config.display_name()] == 0.1
     assert len(eval.evaluator.matchings_by_test_case) != 0
-    assert len(eval.evaluator.matchings_by_test_case[TEST_CASE.name]) == len(TEST_DATA)
+    assert len(eval.evaluator.matchings_by_test_case[config.display_name()]) != 0
```

Comment:
Nitpicking: is it easy to actual use 2 configurations which would lead to different caching results and verify correctness?